getting-started/windows-calico/quickstart.md

```diff
@@ -95,9 +224,9 @@ The following steps install a Kubernetes cluster on a single Windows node, with
    - Downloads Kubernetes binaries.
    - Configures {{site.prodnameWindows}} and starts the Calico service.
 
-1. To skip any of the above tasks, see [Configure installation parameters](#configure-installation-parameters).
+   To skip any of the above tasks, see [Configure installation parameters](#configure-installation-parameters).
 
-1. Run install-calico-windows.ps1 for your datastore with parameters for your implementation.
+   Run install-calico-windows.ps1 for your datastore with parameters for your implementation.
```

Comment:
This wording in this step is a bit confusing. Suggest the following:

1. Install Calico for Windows for your datastore with using the default parameters or [customize installation parameters]. (#configure-installation-parameters).
The powershell script downloads Calico for Windows release binary, Kubernetes binaries, Windows utilities files, configures Calico for Windows, and starts the Calico service.